app/build.gradle

```diff
@@ -12,6 +12,7 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'realm-android'
 
 def YOUTUBE_API_KEY = System.getenv('YOUTUBE_API_KEY') ?: "YOUR_API_KEY"
+def MAPBOX_API_KEY = '"'+System.getenv('MAPBOX_API_KEY')+'"' ?: '"DEFAULT"'
```

Comment:
Extra single and double quotes

Reply:
Sir if we don't do so then it will not return string. In BuildConfig it will show key without string without double quotes.

Reply:
I agree. But, then how is YOUTUBE_API_KEY working?

Reply:
I am not passing this key through menifest so. And this key is unable to fetch in menifest so I thought BuildConfig is better.